stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -211,6 +211,19 @@
 ;;   for the given reward cycle
 (define-map aggregate-public-keys uint (buff 33))
 
+;; State for setting allowances for signer keys to be used in
+;; certain stacking transactions
+(define-map signer-key-authorizations 
+    {
+        signer-key: (buff 33),
+        reward-cycle: uint,
+        period: uint,
```

Comment:
A bit unclear on what "period" means throughout, is this the same as "round"?

Was checking SIP-18 to see if it's related signing structure but don't believe it is...

Reply:
Nvm*** understood that period = # of cycles.